redash/models.py

```diff
@@ -748,7 +750,7 @@ def should_schedule_next(previous_iteration, now, schedule, failures):
 
 class Query(ChangeTrackingMixin, TimestampMixin, BelongsToOrgMixin, db.Model):
     id = Column(db.Integer, primary_key=True)
-    version = Column(db.Integer, default=1)
+    version = Column(db.Integer, default=0)
```

Comment:
No, because defaults only affect new instances, and the bugfixes in `ChangeTrackingMixin` result in the version number being bumped immediately.